srv/components/ha/haproxy/files/haproxy.cfg

```diff
@@ -8,7 +8,7 @@ global
 
     chroot      /var/lib/haproxy
     pidfile     /var/run/haproxy.pid
-    maxconn     {{ 150 // salt['pillar.get']('haproxy:nbproc', 2) }}
+    maxconn     {{ int(0.999 + 150.0 / salt['pillar.get']('haproxy:nbproc', 2)) }}
```

Comment:
This won't work - {{ int() }}. I tried with this change and jinja failed to render with following error:
```
----------
          ID: Setup HAProxy config
    Function: file.managed
        Name: /tmp/haproxy.cfg
      Result: False
     Comment: Unable to manage file: Jinja variable 'int' is undefined
     Started: 15:54:44.946491
    Duration: 187.445 ms
     Changes:   
```
Need to have different logic to round up to the desired value.

Reply:
I guess [|int](https://jinja.palletsprojects.com/en/2.11.x/templates/#int) filter might be used or a temp variable using `{% set ... %}` ...

Reply:
yes. for jinja you cant parse that way..
@t7ko-seagate  you need to have something like `{{ (0.999 + 150.0 / salt['pillar.get']('haproxy:nbproc', 2))| int }}`

Reply:
Fixed.